kedro_azureml/cli_functions.py

```diff
@@ -129,3 +132,11 @@ def verify_configuration_directory_for_azure(click_context, ctx: CliContext):
         )
         if not click.confirm(click.style(msg, fg="yellow")):
             click_context.exit(2)
+
+
+def parse_extra_env_params(extra_env):
+    for entry in extra_env:
+        if not re.match(".+=.*", entry):
```

Comment:
Maybe we should be more strict on the left side, e.g. `[A-Za-z_]+=.*`?

Reply:
Okay! I will allow digits too